paymentsheet/src/test/java/com/stripe/android/paymentsheet/PaymentSheetViewModelTest.kt

```diff
@@ -616,44 +614,18 @@ internal class PaymentSheetViewModelTest {
     }
 
     @Test
-    fun `googlePayState without config should emit Indeterminate`() = runTest {
```

Comment:
These are already tested in `PaymentSheetLoaderTest`. The view model test isn’t the right level to test this anyway because the result from `PaymentSheetLoader` is faked.